content/_data/indexpage/carousel.yml

```diff
@@ -1,14 +1,13 @@
 # 2022 Elections
-- :href: blog/2022/10/20/jenkins-election-announcement/
+- :href: blog/2022/11/17/jenkins-election-candidates-and-results/
   :title: Jenkins Elections 2022
-  :intro: Voting for the Jenkins Board and Officer Elections 2022 begin this Thursday, Nov 17.
-    Voter registration is happening now until Nov 17
+  :intro: The 2022 Jenkins election results are in! There was one nominee for each role, meaning those nominated have been elected!
   :image:
-    :src: images/logos/needs-you/Jenkins_Needs_You-transparent.png
+    :src: images/governance/elections/2022/2022-opengraph.png
```

Comment:
Didn't find where this OG image is stored, any idea?

![image](https://user-images.githubusercontent.com/91831478/202556839-b4edaa51-644c-4e5b-bba4-d05feb272467.png)